exercises/go-counting/canonical-data.json

```diff
@@ -0,0 +1,172 @@
+{
+  "exercise": "go-counting",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Test for black territory on 5x5 board",
+      "property": "territory",
+      "input": {
+        "board": [
+		    "  B  ",
+		    " B B ",
+		    "B W B",
+		    " W W ",
+		    "  W  "
+		],
+        "x": 0,
+        "y": 1
+      },
+      "expected": {
+        "stone": "BLACK",
+        "territory": [[0, 0], [0, 1], [1, 0]]
+      }
+    },
+    {
+      "description": "Test for white territory on 5x5 board",
+      "property": "territory",
+      "input": {
+        "board": [
+		    "  B  ",
+		    " B B ",
+		    "B W B",
+		    " W W ",
+		    "  W  "
+		],
+        "x": 2,
+        "y": 3
+      },
+      "expected": {
+        "stone": "WHITE",
+        "territory": [[2, 3]]
+      }
+    },
+    {
+      "description": "Test for open territory on 5x5 board",
+      "property": "territory",
+      "input": {
+        "board": [
+		    "  B  ",
+		    " B B ",
+		    "B W B",
+		    " W W ",
+		    "  W  "
+		],
+        "x": 1,
+        "y": 4
+      },
+      "expected": {
+        "stone": "NONE",
+        "territory": [[0, 3], [0, 4], [1, 4]]
+      }
+    },
+    {
+      "description": "Test for none territory on 5x5 board",
+      "property": "territory",
+      "input": {
+        "board": [
+		    "  B  ",
+		    " B B ",
+		    "B W B",
+		    " W W ",
+		    "  W  "
+		],
+        "x": 1,
+        "y": 1
+      },
+      "expected": {
+        "stone": "NONE",
+        "territory": []
+      }
+    },
+    {
+      "description": "Test for valid coordinate on 5x5 board",
+      "property": "territory",
+      "input": {
+        "board": [
+		    "  B  ",
+		    " B B ",
+		    "B W B",
+		    " W W ",
+		    "  W  "
+		],
+        "x": -1,
+        "y": 1
+      },
+      "expected": {
+        "stone": "NONE",
+        "territory": []
+      }
+    },
+    {
+      "description": "Test for valid coordinate #2 on 5x5 board",
+      "property": "territory",
+      "input": {
+        "board": [
+		    "  B  ",
+		    " B B ",
+		    "B W B",
+		    " W W ",
+		    "  W  "
+		],
+        "x": 1,
+        "y": 5
+      },
+      "expected": {
+        "stone": "NONE",
+        "territory": []
+      }
+    },
+    {
+      "description": "Test when one territory is the whole board",
+      "property": "territories",
+      "input": {
+        "board": [
+		    " "
+		],
```

Comment:
`,` not necessary.
Try www.jsonlint.com.  It will help point out where the syntax is wrong.